scripts/addMostRecentAutomaticCollection.py

```diff
@@ -0,0 +1,82 @@
+import argparse
+import os
+from uuid import uuid4
+
+import sqlalchemy as sa
+
+from managers import DBManager
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("user")
+    args = parser.parse_args()
+    user = args.user
+
+    dbManager = DBManager(
+        user=os.environ.get("POSTGRES_USER", "localuser"),
+        pswd=os.environ.get("POSTGRES_PSWD", "localpsql"),
+        host=os.environ.get("POSTGRES_HOST", "localhost"),
+        port=os.environ.get("POSTGRES_PORT", "5432"),
+        db=os.environ.get("POSTGRES_NAME", "drb_test_db"),
+    )
+    dbManager.generateEngine()
+    dbManager.createSession()
+
+    uuid = uuid4()
+
+    with dbManager.session.begin() as _session:
+        # First create the collection
+        collection_id = dbManager.session.execute(
+            sa.text(
+                """
+                INSERT INTO collections (
+                  date_created,
+                  uuid,
+                  title,
+                  creator,
+                  description,
+                  owner,
+                  type
+                ) VALUES (
+                  NOW(),
+                  :uuid,
+                  'Most Recent',
+                  :creator,
+                  'Most recently created editions',
+                  :owner,
+                  'automatic'
+                )
+                RETURNING id
+                """
+            ),
+            {
+                "uuid": uuid,
+                "creator": user,
+                "owner": user,
+            },
+        ).scalar()
+
+        # Now the search terms
+        dbManager.session.execute(
+            sa.text(
+                """
+                INSERT INTO automatic_collection (
+                  collection_id,
+                  sort_field,
+                  sort_direction,
+                  "limit"
+                ) VALUES (:collectionId, :sortField, :sortDirection, :limit)
+                """,
+            ),
+            {
+                "collectionId": collection_id,
+                "sortField": "date",
+                "sortDirection": "DESC",
+                "limit": "100",
+            },
+        )
```

Comment:
After these execute statements don't forget to type `dbManager.session.commit` or these changes won't actually occur in the database.

Reply:
You only need to type the commit line one time at the end of this main method.

Reply:
actually, the `with session.begin()` context manager takes care of that! Once the with block is done, the context manager commits, or rolls back if there is an exception

Reply:
Oh that's nice to know. I just looked it up the session API and that's a good thing to remember.